lib/config.js

```diff
@@ -23,22 +22,6 @@ module.exports = (nextConfig = {}) => {
         )
       }
 
-      const {dev, isServer} = options
```

Comment:
We have no CSS to load in Blueprints, this was leftover from Primer CSS's config